<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_QWidget</class>
 <widget class="QWidget" name="main_QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="main_horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="settings_verticalWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>275</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="settings_verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>289</height>
         </size>
        </property>
        <property name="title">
         <string>Camera Connect</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="3">
          <widget class="QDoubleSpinBox" name="mag_doubleSpinBox"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="cam_connect_checkBox">
           <property name="text">
            <string>HW Connect</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="res_mode_doubleSpinBox"/>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="cam_index_doubleSpinBox"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="res_mode_label">
           <property name="text">
            <string>Resolution Mode</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>calibration: (micron/pixel) * Mag.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="calibration_doubleSpinBox">
           <property name="suffix">
            <string> um</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="camera_index_label_2">
           <property name="text">
            <string>Mag.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="camera_index_label">
           <property name="text">
            <string>Camera #</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="4">
          <widget class="QPushButton" name="snap_pushButton">
           <property name="text">
            <string>Snap</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="live_checkBox">
           <property name="text">
            <string>Live</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="defaults_pushButton">
           <property name="text">
            <string>default settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Preview File</string>
        </property>
        <widget class="QLineEdit" name="Job_File_lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>241</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="JobFileBrowseButton">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Job File Path:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="include_preview_checkBox">
         <property name="geometry">
          <rect>
           <x>149</x>
           <y>100</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Include Preview</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_exposure">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>481</height>
         </size>
        </property>
        <property name="title">
         <string>Camera controll</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <item row="16" column="0" colspan="2">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="exposure_GroupBox">
             <property name="title">
              <string>exposure time</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QSlider" name="exposure_horizontalSlider">
                <property name="maximum">
                 <number>2</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="exp_value">
                <property name="maximum">
                 <double>5.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gain_horizontalGroupBox">
             <property name="title">
              <string>gain</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QSlider" name="gain_horizontalSlider">
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="gain_doubleSpinBox">
                <property name="decimals">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="auto_exp_checkBox">
           <property name="text">
            <string>Auto Exposure</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="auto_level_checkBox">
           <property name="text">
            <string>Auto Level display</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
